cvat-ui/src/components/annotation-page/tag-annotation-workspace/tag-annotation-sidebar/shortcuts-select.tsx

```diff
@@ -21,6 +26,81 @@ type Props = {
     labels: Label[];
 };
 
+const componentShortcuts = {
+    SETUP_0_TAG: {
+        name: 'Setup 0 tag',
+        description: 'Setup tag with "label" label',
```

Comment:
"label" label?

Reply:
I suppose description for any value is the same. Something like:

`Create a new tag with corresponding class. The class may be setup in tag annotation sidebar`

Name, also the same for each shortcut:
e.g.: `Create a new tag`